src/pages/home/base/mobile-core/release-notes.md

```diff
@@ -8,6 +8,13 @@ keywords:
 
 # Release notes
 
+## June 17, 2025
+
+### iOS Core 5.5.1
+
+* Fixed an issue causing incorrect rule evaluation of `notContains` and `notEquals` conditions with multiple operands.
```

Comment:
should change the description to be when checking against multiple values, not with multiple operands.